pypeit/spectrographs/keck_deimos.py

```diff
@@ -255,6 +256,9 @@ def config_specific_par(self, scifile, inp_par=None):
             # Since we use the slitmask info to add and remove traces, 'minimum_slit_gap' may undo the matching effort.
             par['calibrations']['slitedges']['minimum_slit_gap'] = 0.
             par['reduce']['slitmask']['assign_obj'] = True
+            # force extraction of undetected objects
+            par['reduce']['slitmask']['force_extract'] = True
+            par['reduce']['slitmask']['force_fwhm'] = 0.5  # arcsec
```

Comment:
force_fwhm is not necessary as discussed above.

Reply:
Removed